src/utils/cookie.ts

```diff
@@ -0,0 +1,23 @@
+import Cookies from 'universal-cookie';
+
+const cookie = new Cookies();
```

Comment:
It'd be better to use localStorage instead of cookies. Here are some reasons for it: https://developer.mozilla.org/en-US/docs/Web/API/Web_Storage_API.

It also means we don't have to install a new dependency.

And here's more on its usage: https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage.

Reply:
Okay, understood. I have made the changes. Could you please test it and let me know?